### PR TITLE
Adding missing required props to foreign and bbmd network port modes

### PR DESCRIPTION
**Pull request by metelik**
Adding missing properties from IPv6 'foreign' an `'bbmd'` modes of network port.

### CODE REVIEW

File: src/bacnet/basic/object/netport.c [hunk 1]
```diff
@@ -1659,6 +1676,354 @@ bool Network_Port_Remote_BBMD_BIP_Lifetime_Set(
...
+{
+    unsigned index = 0; /* offset from instance lookup */
+    bool status = false;
+
+    index = Network_Port_Instance_To_Index(object_instance);
+    if (index < BACNET_NETWORK_PORTS_MAX) {
+        if (Object_List[index].Network_Type == PORT_TYPE_BIP6) {
+            if (addr) {
+              memcpy(addr, Object_List[index].Network.IPv6.BBMD_IP_Address, IP6_ADDRESS_MAX);
+            }
```
Comment (adrigonzo) [inline src/bacnet/basic/object/netport.c:1890]: missing `status = true;`  ?

File: src/bacnet/basic/object/netport.c [hunk 2]
```diff
@@ -1659,6 +1676,354 @@ bool Network_Port_Remote_BBMD_BIP_Lifetime_Set(
...
+}
+
+/**
+ * For a given object instance-number, sets the ip-address
+ * Note: depends on Network_Type being set for this object
+ *
+ * @param  object_instance - object-instance number of the object
+ * @param  addr - pointer to IP6_ADDRESS_MAX = 16 octets buffer
+ *
+ * @return  true if ip-address was set
```
Comment (PeterJeaiem) [inline src/bacnet/basic/object/netport.c:1904]: Can't see anywhere where status is set to anything but false

File: src/bacnet/basic/object/netport.c [hunk 3]
```diff
@@ -1659,6 +1676,354 @@ bool Network_Port_Remote_BBMD_BIP_Lifetime_Set(
...
+bool Network_Port_Remote_BBMD_IP6_Address_Set(
+    uint32_t object_instance, uint8_t *addr)
+{
+    unsigned index = 0; /* offset from instance lookup */
+    bool status = false;
+
+    index = Network_Port_Instance_To_Index(object_instance);
+    if (index < BACNET_NETWORK_PORTS_MAX) {
+        if (Object_List[index].Network_Type == PORT_TYPE_BIP6) {
+            memcpy(Object_List[index].Network.IPv6.BBMD_IP_Address, addr, IP6_ADDRESS_MAX);
```
Comment (adrigonzo) [inline src/bacnet/basic/object/netport.c:1915]: same thing here, status = true

File: src/bacnet/basic/object/netport.c (hunk 1 above)
Comment (metelik) [reply src/bacnet/basic/object/netport.c:1892]: resolved.

File: src/bacnet/basic/object/netport.c (hunk 2 above)
Comment (metelik) [reply src/bacnet/basic/object/netport.c:1906]: resolved.

File: src/bacnet/basic/object/netport.c (hunk 3 above)
Comment (metelik) [reply src/bacnet/basic/object/netport.c:1917]: resolved.

**Review by PeterJeaiem — APPROVED** [review]

**Review by adrigonzo — APPROVED** [review]

**Pull request merged**